.github/workflows/shared_deploy_cloudhub_1_0.yml

```diff
@@ -67,6 +67,7 @@ jobs:
     steps:
       - name: Checkout
         uses: actions/checkout@v4
+        ref: ${{ github.event.workflow_run.head_branch || github.ref }}
```

Comment:
I'm really not sure the workflow can use the `github.event.workflow_run.head_branch` directly here, because this is just the action... but let's see 

I was looking like

From the app, we pass the ref from there to the actions - because the `workflow triggers` that workflow firstly

![image](https://github.com/user-attachments/assets/6a877ab2-47c2-4c1e-b2d1-d7e94c0d778e)

Reply:
Thanks @andyduong1920 , you're right 🙏 

I'll try your approach.

Reference: https://stackoverflow.com/questions/71458189/github-action-workflow-call-does-not-use-up-to-date-input-values